docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -0,0 +1,1255 @@
+        components.</para>
+
+        <para>Some components may use only 1 category, some can use several.
+        The values file can contain more than one storage plane definition for
+        each category. The first storage plane in the list for each category
+        is used as the default location to store that category of data. These
+        categories minimize the exposure of plane data to components that
+        don't need them. For example the ECLCC Server component does not need
+        to know about landing zones, or where Dali stores its data, so it only
+        mounts the plane categories it needs.</para>
+      </sect3>
+
+      <sect3 id="YML_EphemeralStorage">
+        <title>Ephemeral Storage</title>
+
+        <para>Typically, by design cloud systems data storage is ephemeral.
+        Ephemeral storage is allocated when the HPCC Systems cluster is
+        installed and deleted when the chart is uninstalled. This is helpful
+        in keeping cloud costs down but may not be appropriate for your
+        data.</para>
+
+        <para>In your system, you would want to override the delivered stock
+        value(s) with storage appropriate for your specific needs. The
+        supplied values create ephemeral or temporary persistent volumes that
+        get automatically deleted when the chart is uninstalled. You probably
+        want the storage to be persistent. You should customize the storage to
+        a more suitable configuration for your needs.</para>
+      </sect3>
+
+      <sect3 id="YAML_Persist_storage">
+        <title>Persistent Storage</title>
+
+        <para>Kubernetes uses persistent volume claims (pvcs) to provide
+        access to data storage. HPCC Systems supports cloud storage through
+        the cloud provider that can be exposed through these persistent volume
+        claims.</para>
+
+        <para>Persistent Volume Claims can be created by overriding the
+        storage values in the delivered Helm chart. The values in the
+        <emphasis
+        role="boldIt">examples/local/values-localfile.yaml</emphasis> provided
+        override the corresponding entries in the original delivered stock
+        HPCC Systems helm chart. The localfile chart creates persistent
+        storage volumes. You can use the
+        <emphasis>values-localfile.yaml</emphasis> directly (as demonstrated
+        in separate docs/tutorials) or you can use it as a basis for creating
+        your own override chart.</para>
+
+        <para>To define a storage plane that utilizes a PVC, you must decide
+        on where that data will reside. You create the storage directories,
+        with the appropriate names and then you can install the localfiles
+        Helm chart to create the volumes to use the local storage option, such
+        as in the following example:</para>
+
+        <programlisting>helm install mycluster hpcc/ -f examples/local/values-localfile.yaml</programlisting>
```

Comment:
to be consistent, should use repo. form (not point to directory), i.e.: should say "hpcc/hpcc" not "hpcc/"